### ISSUE TITLE
Hostname

### ISSUE THREAD

**Issue by ukbssltd** [vpsserver-com/whmcs-vpsserver-com#4]
When trying to change the host name in the client area it hangs and does nothing just sits their spinning. 

Also from the initial info page, you cant select change password it does nothing you have to go to another tab first i.e hostname then back to change pass. 

I feel this module is poorly made and rushed. 

Lots of bugs.

**Comment by SvenBuis** [comment]
Could you please provide detailed steps and screenshots of this issue, we are not experiencing the same issue.

**Comment by ukbssltd** [comment]
Yes when in the whmcs client area on the server page click change hostname, then try and change it it just brings up a load simble and does nothing. 

With the change password from service info page in client area you click change password and again it does nothing as it takes it to the wrong url it seems.

**Comment by ukbssltd** [comment]
This is still causing a massive headache;

Video of issue here: [](https://assets.ukbss.com/systems/Video-Sun-Sep-12-2021-00-58-30.webm)

**Comment by ukbssltd** [comment]
[https://assets.ukbss.com/systems/Video-Sun-Sep-12-2021-00-58-30.webm](https://assets.ukbss.com/systems/Video-Sun-Sep-12-2021-00-58-30.webm)

**Comment by ukbssltd** [comment]
This is an output form api at vpsserver.com {"url":"\/v1\/servers\/149105.json","code":401,"message":"No identity found. You can skip this check by configuring  `requireIdentity` to be `false`."}

**Comment by ukbssltd** [comment]
I believe after trying multiple fixes and attempts this does not work in api. 

I think it has something to do with the fact the server has to restart after the change and api won't allow this. We think its an api error so will nt work.

Also as you can see above don't expect any response and don't bother with VPS Server.com either as both service and this module you will not get support. its "Self Managed".

**Comment by SvenBuis** [comment]
The API commands works fine from our side, could you please provide the command you are sending that is failing?

The command does indeed not work in the WHMCS module, there is no ETA on a fix for this.
